package.json

```diff
@@ -45,13 +43,11 @@
   "dependencies": {
     "@babel/runtime": "^7.5.5",
     "@testing-library/dom": "^6.0.0",
-    "@types/react-dom": "*"
+    "@types/testing-library__react": "^9.1.0"
```

Comment:
We don't need the react-dom typings as a dep because they'll come along with testing-library__react :)